app/javascript/src/stylesheets/pages/supervisors.scss

```diff
@@ -1,46 +1,74 @@
-$indictator_negative_color: #ff0000;
-$indicator_positive_color: #00c800;
+$indictator_negative_color: #f44336;
+$indicator_positive_color: #4caf50;
 
 .supervisor_indicator_container {
-  width: 20em;
+  display: flex;
 }
 
 .supervisor_indicator_legend {
-  font-size: x-small;
+  display: flex;
+  align-items: center;
+  font-size: 1rem;
+  margin-bottom: 0.25rem;
 }
 
 .supervisor_indicator_legend_negative,
 .supervisor_indicator_legend_positive {
-  border: 1px solid;
   font-size: smaller;
   padding: .25em;
-  width: 50%;
+  flex-grow: 1;
 }
 
 .supervisor_indicator_negative,
 .supervisor_indicator_legend_negative {
   background: $indictator_negative_color;
-  border: 1px solid;
   float: right;
   text-align: center;
+  border-bottom-right-radius: 0.25em;
+  border-top-right-radius: 0.25em;
 }
 
 .supervisor_indicator_positive,
 .supervisor_indicator_legend_positive {
   background: $indicator_positive_color;
-  border: 1px solid;
   float: left;
```

Comment:
floats don't do anything with flex

Reply:
Will do. Yeah I was planning on doing a cleanup pass. I wanted to make sure that I was working the right direction first.